spec/furniture/marketplace/cart/delivery_area_component_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+RSpec.describe Marketplace::Cart::DeliveryAreaComponent, type: :component do
+  subject(:output) { render_inline(component) }
+
+  let(:component) { described_class.new(cart: cart) }
+
+  let(:cart) { create(:marketplace_cart, :with_products, marketplace: marketplace) }
```

Comment:
Why does the cart need products?

Reply:
Ah good catch. I got confused for a moment and thought I needed this to display stuff (that was outside the component scope, actually) and forgot to remove it.